integration/generic-metadata/hero.ts

```diff
@@ -287,9 +287,9 @@ export const Villain = {
 };
 
 export interface HeroService {
-  FindOneHero(request: HeroById, metadata?: Foo): Promise<Hero>;
```

Comment:
I think the `metadataType=Foo...` should probably still be used here/take precedence over `addGrpcMetadata`? Or how do you think using both options at the same time should behave?

Reply:
I completely missed `metadataType=Foo` parameter. I think there should be `metadata?: Foo`, I'll fix that and probably add another test case